pushnotifications/src/main/java/com/pusher/pushnotifications/api/PushNotificationsAPI.kt

```diff
@@ -309,6 +319,15 @@ class PushNotificationsAPI(private val instanceId: String, overrideHostURL: Stri
 
         throw PushNotificationsAPIBadJWT("Unknown reason")
       }
+      if (response.code() == 422) {
+        val responseErrorBody = response?.errorBody()
+        if (responseErrorBody != null) {
+          val error = safeExtractJsonError(responseErrorBody.string())
+          throw PushNotificationsAPIUnprocessableEntity("${error?.error}: ${error?.description}")
```

Comment:
what does the error.error and error.description actually look like? Does it inform people that their issue is they have 10 registered devices already?

Reply:
Yes, it will be as follows:
> User has reached the maximum of 10 devices